src/test/Calculator.test.js

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react';
+import CalculatorContainer from '../components/calculator';
+
+it('test calculator component', () => {
+  const tree = render(<CalculatorContainer />);
+  expect(tree).toMatchSnapshot();
+});
+
+it('test calculator click event', async () => {
+  const { getByTestId } = render(<CalculatorContainer />);
+
+  expect(getByTestId('txt-area').textContent).toBe('0');
```

Comment:
- [ ] Excellent use of Jest snapshots testing 👏🏻  But you need to use Jest snapshots to test all the components. Actually, I only see "Calculator.test.js.snap,"  Please make sure you add Jest snapshots for testing these components:  `App.js, Home.js, Quotes.js, and Navbar.js`. 👨🏻‍🎨